client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/TasksClientDocumentationIT.java

```diff
@@ -67,7 +67,7 @@ public void testListTasks() throws IOException {
             // tag::list-tasks-request-filter
             request.setActions("cluster:*"); // <1>
             request.setNodes("nodeId1", "nodeId2"); // <2>
-            request.setParentTaskId(new TaskId("parentTaskId", 42)); // <3>
+            request.setTargetParentTaskId(new TaskId("parentTaskId", 42)); // <3>
```

Comment:
Ah unfortunately this is in the HLRC API so we can't change it so freely. We can introduce `setTargetParentTaskId` and deprecate the older method, that should be enough.

Reply:
The HLRC is not supported in 8x. So we should be fine to change this? I'd imagine all this code is getting removed at some point.

Reply:
I have added back the older methods, now deprecated.